tests/session/session-startTransaction_error-003.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+MongoDB\Driver\Session::startTransaction() with wrong argument for options array
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_server_version('<', '4.0'); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(URI);
+$session = $manager->startSession();
+
+$options = [
+    2,
+    new stdClass,
+];
+
+foreach ($options as $txnOptions) {
+    $session->startTransaction($txnOptions);
+}
+
+?>
+===DONE===
+<?php exit(0); ?>
+--EXPECTF--
+Warning: MongoDB\Driver\Session::startTransaction() expects parameter 1 to be array, integer given in %s%session-startTransaction_error-003.php on line %d
```

Comment:
Other tests simply use `in %s on line %d`. Any reason not to do the same here and make the output a bit more robust?

Reply:
ah, crap. That's what you get doing it different in your other major (Xdebug) project. Will fix (again).